h2o-core/src/main/java/hex/grid/filter/DefaultPermutationFilter.java

```diff
@@ -0,0 +1,38 @@
+package hex.grid.filter;
+
+import hex.Model;
+
+import java.util.List;
+
+public class DefaultPermutationFilter<MP extends Model.Parameters> implements PermutationFilter<MP> {
+
+  private List<PermutationFilterFunction<MP>> _filterFunctions;
+
+  public DefaultPermutationFilter(List<PermutationFilterFunction<MP>> filterFunctions) {
```

Comment:
You could provide a constructor with varargs to make it easier to use.

Reply:
@seb-h2o yep, that would be even better, thx